lib/dependency_container.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:dio/dio.dart';
+import 'package:get_it/get_it.dart';
+import 'package:marvelapp_flutter/data/data_sources/local/dao/character_dao.dart';
+import 'package:marvelapp_flutter/data/data_sources/local/database/database.dart';
+import 'package:marvelapp_flutter/data/data_sources/local/character_data_source.dart';
+import 'package:marvelapp_flutter/data/data_sources/remote/marvell_api_client.dart';
+import 'package:marvelapp_flutter/data/data_sources/remote/remote_data_source.dart';
+import 'package:marvelapp_flutter/data/repository/default_marvell_repository.dart';
+import 'package:marvelapp_flutter/domain/use_cases/get_character_use_case.dart';
+import 'package:marvelapp_flutter/domain/use_cases/get_characters_use_case.dart';
+import 'package:marvelapp_flutter/domain/use_cases/get_series_with_character_use_case.dart';
+
+final getIt = GetIt.instance;
+
+void setupDependences() {
```

Comment:
```suggestion
void setupDependencies() {
```

Reply:
thank you. i think about it. IDE show it, but i not understand in that moment

Reply:
done